processes/security_team_onboarding.md

```diff
@@ -25,7 +25,7 @@ Patiently wait for feedback and +1’s from the team and a reasonable time to en
 Once acceptance has been acquired by group members, the following should take place to setup the new member:
 
 * Confirm that the new member has read, understands, and agrees to uphold the
-  [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md).
+  [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md). The work of triaging vulnerabilities at the Security WG is about handling sensitive matters with open source maintainers and communication should be handled with care. If in doubt on how to communicate reach out to the Moderation WG team and ask for guidance. As a member of the Security WG please also be aware of public statements your make and interactions.
 * Mentor should ping @jasnell, @gibfahn, @cjihrig or @mhdawson to add the new member to the repository.
```

Comment:
Unrelated to this PR, but this list should likely be updated. Based on the current membership list, I would drop James and Gibson, and add @sam-github and @ChALkeR (assuming they're OK with it).

Reply:
+1 from me

Reply:
please confirm @sam-github and @ChALkeR

Reply:
I'll open a follow-up PR for this